### ISSUE TITLE
got zero models from mmcif written from pymol selection, expected one model

### ISSUE THREAD

**Issue by diogomart** [project-gemmi/gemmi#327]
Hello,

The mmcif attached below, written from pymol, parses to a structure with zero models. Expected one model. Parsing the same file with prody results in a data structure with a model. Reading it again with pymol displays the atoms.

```python
>>> g = gemmi.read_structure("from_pymol.cif")
>>> print(g)

<gemmi.Structure 4fkg with 0 model(s)>
```

<details>
    <summary>from_pymol.cif</summary>

```
# generated by PyMOL 2.5.0
#
data_4fkg
_entry.id 4fkg
#
_cell.entry_id 4fkg
_cell.length_a 53.305
_cell.length_b 71.634
_cell.length_c 72.407
_cell.angle_alpha 90.00
_cell.angle_beta  90.00
_cell.angle_gamma 90.00
_symmetry.entry_id 4fkg
_symmetry.space_group_name_H-M 'P 21 21 21'
#
loop_
_atom_site.group_PDB
_atom_site.id
_atom_site.type_symbol
_atom_site.label_atom_id
_atom_site.label_alt_id
_atom_site.label_comp_id
_atom_site.label_asym_id
_atom_site.label_entity_id
_atom_site.label_seq_id
_atom_site.pdbx_PDB_ins_code
_atom_site.Cartn_x
_atom_site.Cartn_y
_atom_site.Cartn_z
_atom_site.occupancy
_atom_site.B_iso_or_equiv
_atom_site.pdbx_formal_charge
_atom_site.auth_asym_id
_atom_site.pdbx_PDB_model_num
ATOM   1   N N   . LYS A 1 89 ?  9.327 -12.007 -23.312 1.00  24.18 0 A 1
ATOM   2   C CA  . LYS A 1 89 ? 10.222 -12.541 -24.329 1.00  24.85 0 A 1
ATOM   3   C C   . LYS A 1 89 ? 11.629 -11.974 -24.179 1.00  26.62 0 A 1
ATOM   4   O O   . LYS A 1 89 ? 12.596 -12.735 -24.287 1.00  28.52 0 A 1
ATOM   5   C CB  . LYS A 1 89 ?  9.650 -12.338 -25.738 1.00  27.40 0 A 1
ATOM   6   C CG  . LYS A 1 89 ? 10.520 -12.948 -26.839 1.00  45.43 0 A 1
ATOM   7   C CD  . LYS A 1 89 ?  9.934 -12.790 -28.223 1.00  53.19 0 A 1
ATOM   8   C CE  . LYS A 1 89 ? 10.829 -13.380 -29.280 1.00  60.75 0 A 1
ATOM   9   N NZ  . LYS A 1 89 ? 10.245 -13.221 -30.635 1.00  70.50 0 A 1
ATOM   10  N N   . MET A 1 91 ? 13.037 -10.911 -21.352 1.00  23.41 0 A 1
ATOM   11  C CA  A MET A 1 91 ? 13.690 -11.518 -20.194 0.50  22.27 0 A 1
ATOM   12  C CA  B MET A 1 91 ? 13.613 -11.530 -20.150 0.50  22.36 0 A 1
ATOM   13  C C   . MET A 1 91 ? 14.171 -12.922 -20.522 1.00  28.37 0 A 1
ATOM   14  O O   . MET A 1 91 ? 15.281 -13.300 -20.106 1.00  28.74 0 A 1
ATOM   15  C CB  A MET A 1 91 ? 12.770 -11.531 -19.000 0.50  23.07 0 A 1
ATOM   16  C CB  B MET A 1 91 ? 12.526 -11.668 -19.073 0.50  23.32 0 A 1
ATOM   17  C CG  A MET A 1 91 ? 12.679 -10.187 -18.344 0.50  24.33 0 A 1
ATOM   18  C CG  B MET A 1 91 ? 12.302 -10.389 -18.267 0.50  24.99 0 A 1
ATOM   19  S SD  A MET A 1 91 ? 12.088 -10.379 -16.674 0.50  27.44 0 A 1
ATOM   20  S SD  B MET A 1 91 ? 10.663 -10.192 -17.478 0.50  27.98 0 A 1
ATOM   21  C CE  A MET A 1 91 ? 10.473 -10.990 -17.025 0.50  23.02 0 A 1
ATOM   22  C CE  B MET A 1 91 ? 10.250 -11.739 -17.208 0.50  26.08 0 A 1
ATOM   23  N N   . ASP A 1 92 ? 13.369 -13.700 -21.272 1.00  24.93 0 A 1
ATOM   24  C CA  . ASP A 1 92 ? 13.742 -15.057 -21.682 1.00  26.20 0 A 1
ATOM   25  C C   . ASP A 1 92 ? 15.001 -14.994 -22.553 1.00  32.66 0 A 1
ATOM   26  O O   . ASP A 1 92 ? 15.937 -15.789 -22.346 1.00  34.34 0 A 1
ATOM   27  C CB  . ASP A 1 92 ? 12.593 -15.729 -22.466 1.00  28.34 0 A 1
ATOM   28  C CG  . ASP A 1 92 ? 11.437 -16.264 -21.632 0.80  35.60 0 A 1
ATOM   29  O OD1 . ASP A 1 92 ? 11.559 -16.293 -20.383 1.00  36.22 0 A 1
ATOM   30  O OD2 . ASP A 1 92 ? 10.386 -16.598 -22.221 0.80  38.42 0 A 1
```

</details>

**Comment by wojdyr** [comment]
Hi, it was just fixed this month in #325

**Comment by diogomart** [comment]
Awesome.
Sorry for the duplication.
